### PR TITLE
[Squid][SMB][Automation]Add scenarios

### PR DESCRIPTION
**Pull request by MohitBis**
# Description

Added below scenarios automation.
1. Verify IO runs over cifs mount
2. Validate SMB share options are getting overwritten after restarting cephfs mds service
3. Validate SMB share options are getting overwritten after restarting smb service

### CODE REVIEW

File: tests/smb/smb_service_restart.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        )
+
+        # Change smb share option using spec
+        generate_apply_smb_spec(installer, file_type, smb_spec, file_mount)
+
+        # Restart services
+        out = CephAdm(installer).ceph.orch.restart(restart_service)
+        if "Scheduled" not in out:
+            raise OperationFailedError(f"Fail to restart {restart_service}")
+        sleep(10)
```
Comment (adityaramteke) [inline tests/smb/smb_service_restart.py:110]: @MohitBis can we wait for services to come up instead adding sleep here? If service does not come up then the below check will fail.

**Review by adityaramteke — CHANGES_REQUESTED** [review]

File: tests/smb/smb_service_restart.py (hunk 1 above)
Comment (MohitBis) [reply tests/smb/smb_service_restart.py]: Thanks @adityaramteke 

Updated

**Comment by MohitBis** [comment]
Latest Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-CZXMDK/

**Review by adityaramteke — APPROVED** [review]
LGTM

**Review by pranavprakash20 — APPROVED** [review]
Lgtm

**Review by vdas-redhat — APPROVED** [review]
LGTM

**Pull request merged**